lib/nessus/report_item.rb

```diff
@@ -102,6 +104,23 @@ def method_missing(method, *args)
         end
       end
 
+      # older versions of Nessus use <vpr_score> while newer versions of Nessus 
+      #   use <vulnerability_priority_rating>. This allows either tag to be 
+      #   pulled in to the vpr_score mapping
+      if method_name.starts_with?('vpr')
```

Comment:
This will catch any fields that start with `vpr`? Are there anymore besides `vpr_score`? If not then we can simply just compare it exactly:
```suggestion
      if method_name == 'vpr_score'
```